src/golang/cmd/server/handler/configure_storage.go

```diff
@@ -0,0 +1,140 @@
+package handler
+
+import (
+	"context"
+	"net/http"
+	"path"
+
+	"github.com/aqueducthq/aqueduct/cmd/server/routes"
+	"github.com/aqueducthq/aqueduct/config"
+	aq_context "github.com/aqueducthq/aqueduct/lib/context"
+	"github.com/aqueducthq/aqueduct/lib/database"
+	"github.com/aqueducthq/aqueduct/lib/models/shared"
+	"github.com/aqueducthq/aqueduct/lib/repos"
+	"github.com/aqueducthq/aqueduct/lib/workflow/utils"
+	"github.com/dropbox/godropbox/errors"
+	"github.com/go-chi/chi/v5"
+	"github.com/google/uuid"
+	log "github.com/sirupsen/logrus"
+)
+
+// Route: /config/storage/{integrationID}
+// Method: POST
+// Request:
+//
+//	Headers:
+//		`api-key`: user's API Key
+//
+// Response: none
+type ConfigureStorageHandler struct {
+	PostHandler
+
+	Database database.Database
+
+	ArtifactRepo       repos.Artifact
+	ArtifactResultRepo repos.ArtifactResult
+	DAGRepo            repos.DAG
+	IntegrationRepo    repos.Integration
+	OperatorRepo       repos.Operator
+
+	PauseServer   func()
```

Comment:
What's the go standard to call a function pointer? I personally feel `PauseServerFn` is easier to understand

Reply:
Sure, addressed